src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -817,6 +817,7 @@
             "CurrentThermostatSuggestion",
             "ThermostatSuggestionNotFollowingReason"
         ],
+        "Time Format Localization": ["ClusterRevision"],
```

Comment:
I guess this does not list the other non-list attributes because we use the ZAP RAM store for the featuremap and whatnot?

Reply:
Yes, I wasn't able to remove more elements since we still need the values from ZAP, even for the two attributes where we need them for the "default" value just in case.